Assets/HoloToolkit/Input/Scripts/Utilities/Interactions/HandDraggable.cs

```diff
@@ -293,6 +297,10 @@ public void StopDragging()
             isDragging = false;
             currentInputSource = null;
             currentInputSourceId = 0;
+            if (hostRigidbody != null)
+            {
+                hostRigidbody.isKinematic = false;
```

Comment:
What if `isKinematic` was true to begin with?

Reply:
That's a fair point, if the hand draggable object was never meant to be affected, then this would change it's behaviour in the scene. Nice catch @keveleigh 

@michaellperry two suggestions:
1: Make this configurable, so the user can opt to have the IsKinematic enabled / disabled with this option (Name it something like "FreezePhysicsWhenGrabbed").  Only do the behaviour if enabled.
2: Alter the flow of the change to:
* Have a private "Altered" bool to denote the setting was changed
* Only set  "Altered" bool to true if isKinematic was altered (e.g. from true to false)
* Only set isKinematic back to true of the "Altered" bool was true (and reset "Altered" back to false)